apps/src/shared/gamma/containers/bottom-tabs/BottomTabsContainer.tsx

```diff
@@ -2,34 +2,34 @@ import React from 'react';
 import type { NativeSyntheticEvent } from 'react-native';
 import {
   BottomTabs,
+  BottomTabsProps,
   BottomTabsScreen,
   BottomTabsScreenProps,
   NativeFocusChangeEvent,
 } from 'react-native-screens';
-import { Colors } from '../../../styling/Colors';
 import ConfigWrapperContext from './ConfigWrapperContext';
 
 export interface TabConfiguration {
   tabScreenProps: BottomTabsScreenProps;
   component: React.ComponentType;
 }
 
-export interface BottomTabsContainerProps {
-  tabConfigs: TabConfiguration[];
-}
+// Currently assumes controlled bottom tabs
+export type BottomTabsContainerProps = Omit<BottomTabsProps, 'experimentalControlNavigationStateInJS' | 'onNativeFocusChange'> & { tabConfigs: TabConfiguration[]; }
```

Comment:
I don't think removing these types is necessary. It might be always nice to pass them during testing. This unnecessarily restricts this possibility.